backend/src/feedback/generateLink.ts

```diff
@@ -1,19 +1,54 @@
+import { nanoid } from 'nanoid';
+import mail from '@sendgrid/mail';
 import { MutationResolvers } from '../graphql/resolvers-types';
 import { getOrMakeUser } from '../user/getOrMakeUser';
-import { uniqueId } from '../utils/uniqueId';
+import { EMAIL_SENDER, FEEDBACK_TEMPLATE_ID, REPLY_TO } from '../utils/constants';
+import { isDevEnv } from '../utils/envTools';
+import { feedbackUniqueId } from './utils/feedbackUniqueId';
 
 export const generateLink: MutationResolvers['generateLink'] = async (
   parent,
   { data: { email } },
   { prisma },
 ) => {
-  const user = await getOrMakeUser({ email });
+  try {
+    const user = await getOrMakeUser({ email });
 
-  await prisma.feedbackBase.create({
-    data: { uid: uniqueId(), user: { connect: { id: user.id } } },
-  });
-  return {
-    // TODO add error handling
-    success: true,
-  };
+    const feedbackUid = feedbackUniqueId();
+    const reportUid = nanoid();
```

Comment:
Can't we use the feedbackUniqueId format for reports, instead of using he default nanoid?
They are formatted to be readable, like google meet urls.

Reply:
This one is the report link, isn't it supposed to be accessible only via email link?